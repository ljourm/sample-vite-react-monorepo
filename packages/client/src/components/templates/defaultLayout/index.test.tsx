import { render } from "@testing-library/react";
import Component from "./";

test("snapshot", () => {
  const { asFragment } = render(
    <Component>
      <p>content</p>
    </Component>
  );
  expect(asFragment()).toMatchSnapshot();
});
