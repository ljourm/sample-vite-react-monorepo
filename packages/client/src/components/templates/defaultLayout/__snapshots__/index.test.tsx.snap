// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<DocumentFragment>
  <div
    class="wrapper"
  >
    <header>
      <img
        alt="product logo"
        class="product-logo"
        src=""
      />
      <ul>
        <li>
          <a
            href="/"
          >
            ほーむ
          </a>
        </li>
        <li>
          <a
            href="/about/"
          >
            あばうと
          </a>
        </li>
      </ul>
    </header>
    <main>
      <p>
        content
      </p>
    </main>
  </div>
</DocumentFragment>
`;
